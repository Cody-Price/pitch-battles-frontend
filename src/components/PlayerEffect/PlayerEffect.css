.player-effect {
  animation: player-claw steps(8) 1s;
  -webkit-animation: player-claw steps(8) 1s;
  -moz-animation: player-claw steps(8) 1s;
  -o-animation: player-claw steps(8) 1s;

  width: 192px;
  height: 192px;
  position: absolute;
  top: 40px;
  left: 0px;
  background-size: 1536px 192px;
}

@keyframes player-claw {
  100% {
    background-position: -1536px;
  }
}
