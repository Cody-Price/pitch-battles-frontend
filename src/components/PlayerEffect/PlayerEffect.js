import React, { Component } from "react";

import "./PlayerEffect.css";

class PlayerEffect extends Component {
  render() {
    return <div className="player-effect" />;
  }
}

export default PlayerEffect;
