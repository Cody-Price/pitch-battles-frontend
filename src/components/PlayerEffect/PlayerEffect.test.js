import React from "react";
import ReactDOM from "react-dom";
import PlayerEffect from "./PlayerEffect";
import Enzyme, { shallow } from "enzyme";
const EnzymeAdapter = require("enzyme-adapter-react-16");

Enzyme.configure({ adapter: new EnzymeAdapter() });

describe("PlayerEffect", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<PlayerEffect />);
  });

  it("should match the snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
