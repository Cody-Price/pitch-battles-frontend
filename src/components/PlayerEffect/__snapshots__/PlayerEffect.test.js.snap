// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayerEffect should match the snapshot 1`] = `
<div
  className="player-effect"
/>
`;
